File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/php/Manusia.php
<?php

	class Manusia{
		
		var $nama;
		var $alamat;
		var $no_ktp;
		var $no_telepon;
		
		function Manusia(){
		}
		
		function setNama($nama){
			$this->nama = $nama;
		}
		
		function getNama(){
			return $this->nama;
		}
		
		function setAlamat($alamat){
			$this->alamat = $alamat;
		}
		
		function getAlamat(){
			return $this->alamat;
		}
		
		function setNoKTP($no_ktp){
			$this->no_ktp = $no_ktp;
		}
		
		function getNoKTP(){
			return $this->no_ktp;
		}
		
		function setNoTelepon($no_telepon){
			$this->no_telepon = $no_telepon;
		}
		
		function getNoTelepon(){
			return $this->no_telepon;
		}
		
		
		
	}

?>

File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/cpp/Manusia.cpp
class Manusia{
	
	private:
		string nama;
		string alamat;
		string no_ktp;
		string no_telepon;
	
	public:
		Manusia(){
		}
		
		void setNama(string nama){
			this->nama = nama;
		}
		
		string getNama(){
			return this->nama;
		}
		
		void setAlamat(string alamat){
			this->alamat = alamat;
		}
		
		string getAlamat(){
			return this->alamat;
		}
		
		void setNoKTP(string no_ktp){
			this->no_ktp = no_ktp;
		}
		
		string getNoKTP(){
			return this->no_ktp;
		}
		
		void setNoTelepon(string no_telepon){
			this->no_telepon = no_telepon;
		}
		
		string getNoTelepon(){
			return this->no_telepon;
		}
		
		~Manusia(){
		}
	
};